### PR TITLE
feat: Update to release version of decorators

### PR DESCRIPTION
**Pull request by lfvbarcus**
This PR updates dependency to reqstool-python-decorators to an official version. A release of this repository will be done after merge.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.10"
 poetry = "^1.7.1"
-reqstool-python-decorators = "^0.1.dev6" #Change later when proper release is made.
```
Comment (lfvjimisola) [inline pyproject.toml:29]: We should set an explicit version (did we do that with hatch?) and not a range.

See: https://python-poetry.org/docs/dependency-specification/#caret-requirements

Comment (lfvbarcus) [reply pyproject.toml:29]: Yes, we are targeting specific versions in the Hatch plugin. This is now fixed here as well

Comment (lfvdavid) [reply pyproject.toml:29]: Think we can close this PR as the version has been updated in PR #12

**Comment by lfvdavid** [comment]
Highjacked this PR to update to latest release of decorators.

Ready for review.

**Review by lfvjimisola — APPROVED** [review]

**Pull request merged**